packages/blocks/src/index.ts

```diff
@@ -60,7 +60,7 @@ if (env[importIdentifier] === true) {
 }
 
 if (typeof window === 'undefined') {
-  throw new Error(
+  console.error(
```

Comment:
I think it is indeed an error, we shouldn't just ignore it.

Reply:
In the latest next.js, even with the use of client mode, it still have some server-side rendering's works, so throwing exceptions here will cause build(npm run build) errors. `npm run dev` can work, but there are also error reminders

Do you have a good way to solve the build errors  in next.js ? We can discuss it .

Blocksuite cannot be rendered on the server, I think this reminder in console is enough, We also can add a notice in the doc.  Or do you have any other good ideas？

![image](https://github.com/toeverything/blocksuite/assets/5779513/a40c7563-91ee-498d-8ce7-9041c38d105c)

Reply:
Yeah, the error here is correct. Your code here is still not correct.

'use client' also means it has SSR.

Don't hide the error. Next.js intends to run the code in a node.js environment

Reply:
@himself65 
I don't understand Why my code is incorrect. 

In the next.js app router mode, the `use client` is used to represent the client : https://nextjs.org/docs/app/building-your-application/rendering/client-components

How to write the correct code   in next.js's app router mode ？

<img width="1433" alt="Screenshot 2023-09-19 at 02 22 59" src="https://github.com/toeverything/blocksuite/assets/5779513/a7d97a8b-508b-4de5-81bd-f9174311adc2">

Reply:
https://nextjs.org/docs/app/building-your-application/optimizing/lazy-loading#skipping-ssr